crates/chronicle/src/bootstrap/mod.rs

```diff
@@ -396,13 +396,19 @@ where
             info!("Loaded import data from {:?}", path.display());
             contents
         } else {
+            eprintln!("Attempting to import data from stdin ... (Ctrl-D to finish)");
```

Comment:
:+1: This is exactly where `atty::is(Stream::Stdin)` might help so we don't print the message if we're piping from a file/stream.

Reply:
It's a cool idea. And certainly we do in further iterations, along with @ryanbtp's idea to chunk the input stream. We don't currently support importing data interactively, though.

Reply:
`atty::is(Stream::Stdin)` does not work for this purpose, but `grep_cli` does. See https://github.com/softprops/atty/issues/52#issuecomment-886751935.

Reply:
Okay, I think I've figured out what you meant now.